tests/e2e/map/map.spec.js

```diff
@@ -1635,12 +1635,31 @@ test.describe('Map interactions', () => {
         })
       })
 
+      await page.route(/colormaps\/GHRSST_L4_MUR_Sea_Surface_Temperature/, async (route) => {
```

Comment:
Why were these routes added? They are never being fulfilled by test actions

Reply:
They will be.

Reply:
@trevorlang No, they were being used. I just forgot to set the correct json to the correct route. So technically, we could set the same colorMapJson to all 3 different colormaps, but we need to set them. So I could remove the extra .json body files and keep them pointed to the same json as the original body file, or add the actual values like I'm doing now.

Reply:
@macrouch So basically, those actions are mocking the requests that grab the colormap jsondata. If those calls aren't made, they result in errors being thrown in the getColorMap action which leads to a red banner popping up and a failure in the playwright tests.

Reply:
I'm confused, are the tests still only working with two collections? If so, why are so many colormaps being loaded? I don't think we are showing more than one colormap at a time, so we shouldn't be loading more than one colormap

Reply:
@trevorlang explained that you are loading multiple colormaps incase the user switches projections and a different colormap exists for the new project, it will already be loaded. So that explains why the request mocks are needed, but that use case is not covered in the tests, which I think was adding to the confusion over these changes

Reply:
@macrouch @trevorlang That is the reason multiple colormaps are being loaded, however, I'm not sure how to switch projections. The bigger thing is making sure that all colormaps that are part of a collection's gibs tags are loaded correctly. If they aren't, there will be a failure.